semantic-ui/src/test/scala/react/semanticui/collections/menu/MenuHeaderSuite.scala

```diff
@@ -6,7 +6,7 @@ package react.semanticui.collections.menu
 import japgolly.scalajs.react.test._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.common.syntax.vdom._
-import react.semanticui.tagOf2AsC
+import react.semanticui.As.tagOf2AsC
```

Comment:
Actually if this PR fixes the implicit scope we can probably remove these imports. Thanks to Raúl for the smart fix instead of my dumb one 😳

Reply:
That's right, let's remove them